setup.py

```diff
@@ -93,5 +93,5 @@ def run(self):
     },
     packages=packages,
     ext_modules=[Extension('_foo', ['stub.cc'])],
-    scripts=['visualdl/server/visualDL', 'demo/vdl_scratch.py'],
+    scripts=['visualdl/server/visualDL', 'demo/vdl_create_scratch_log'],
```

Comment:
Is there a way to make these .py files using symbolic linking without breaking the non .py call? Making them non .py files is an anti-pattern